internal/handlers/organization.go

```diff
@@ -117,6 +116,19 @@ func (api *API) CreateOrganization(c *gin.Context) {
 		}
 		return
 	}
+
+	// Create a default security group for the organization
+	sg, err := api.createDefaultSecurityGroup(ctx, org.ID.String())
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, models.NewApiInternalError(err))
+	}
+
+	// Update the org with the new security group id
+	if err := api.updateDefaultOrganizationSecGroupId(ctx, sg.ID, org.ID); err != nil {
```

Comment:
If this fails, we should probably delete the security group. Otherwise, we've got a group pointing to this org, but the org doesn't point back. It's an unexpected inconsistent state.

I'm pretty sure you get this rollback for free if you move these security group operations into the same transaction as creating the organization above. I think we'd want all of this in the same transaction anyway.